source/Cosmos.Core/Multiboot/Multiboot2.cs

```diff
@@ -1,117 +1,123 @@
-﻿/*
-* PROJECT:          Cosmos Development
-* CONTENT:          Multiboot2 class
-* PROGRAMMERS:      Valentin Charbonnier <valentinbreiz@gmail.com>
-* RESOURCES:        https://www.gnu.org/software/grub/manual/multiboot2/multiboot.html
-*/
-
-using Cosmos.Core.Multiboot.Tags;
-using IL2CPU.API.Attribs;
-
-namespace Cosmos.Core.Multiboot
-{
-    /// <summary>
-    /// Multiboot2 class. Used for multiboot parsing.
-    /// </summary>
-    public unsafe class Multiboot2
-    {
-        #region Properties
-
-        public static BasicMemoryInformation* BasicMemoryInformation { get; set; }
-        public static bool IsVBEAvailable => Framebuffer->Address != 753664; // Some kinda default number.
-        public static Framebuffer* Framebuffer { get; set; }
-        public static MemoryMap* MemoryMap { get; set; }
+﻿/*

```

Comment:
Please undo the formatting changes to make reviewing easier.

Reply:
Sorry.